### ISSUE TITLE
Extension: innsight

### ISSUE THREAD

**Issue by nkoenen** [mlverse/torch#740]
Hi everyone, 

Our package [`innsight`](https://github.com/bips-hb/innsight) is now available on CRAN (see [here](https://CRAN.R-project.org/package=innsight)) and would be a nice extension for the torch ecosystem. I refer to https://torch.mlverse.org/packages/

>  If you are developing a torch extension, contact us on GitHub so we can feature it! We welcome all kinds of extensions: datasets, models, or any kind of functionality built on torch!  

The `innsight` package first converts a neural network (based on packages `torch`, `keras` or `neuralnet`) internally into a torch model, which can then be used to interpret or explain individual predictions. Implemented are some methods like 'Gradient', 'Gradient x Input', 'SmoothGrad', 'LRP', 'DeepLift' (hopefully more soon). Of course, the package is still in an early development phase, but I would be very happy if it gets included in the torch ecosystem already.

**Comment by dfalbel** [comment]
Hey @nkoenen ! 

Thanks for reaching out, `innsight` looks really great! Can't wait for trying it out!
I just added it to the ecosystem gallery with https://github.com/mlverse/torch/commit/d7e98379617b510be362bb7d8da1cb0c7227908d

Let me know if you want to to publish a release blog post in the [RStudio AI blog](https://blogs.rstudio.com/ai/) also!

**Comment by nkoenen** [comment]
Hey @dfalbel,
I just uploaded a logo for my package. Maybe you can add it to the torch ecosystems page to keep it nice and aesthetic :)

**Comment by dfalbel** [comment]
Hey @nkoenen ! Just added the hex-logo to the website!
